src/lib/__tests__/routing/spa-replace-state-test.ts

```diff
@@ -20,9 +20,10 @@ describe('routing using `replaceState`', () => {
     // -- Flow
     // 1. Initial: '/'
     // 2. Refine: '/?indexName[query]=Apple'
-    // 3. Dispose: '/about'
-    // 4. Route change (with `replaceState`): '/about'
-    // 5. Back: '/about'
+    // 3. Dispose: does not yet write
+    // 4. Route change (with `replaceState`): '/about?external=true', replaces state 2
+    // 3. Dispose: writes '/about' (this is a bug, and should be fixed when we have a way to prevent it)
+    // 5. Back: '/about?external=true'
```

Comment:
```suggestion
    // 5. Dispose: writes '/about' (this is a bug, and should be fixed when we have a way to prevent it)
    // 6. Back: '/about?external=true'
```